### ISSUE TITLE
How to check Testing accuracy

### ISSUE THREAD

**Issue by marly892** [you359/Keras-FasterRCNN#80]
Hi! thanks for you code, it is very useful as an introduction to Faster-RCNN.

I have trained the model on my dataset and I tested it producing corresponding images in the "results_imgs" folder. I wonder, which metrics I can use to test numerically the accuracy of the classification and bounding boxes predictions on the test set? Is there a part of the code I am missing?

thanks a lot
